libs/features/components/banners/src/lib/components/team-member/team-member.component.ts

```diff
@@ -0,0 +1,42 @@
+    description : 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero quia quae aspernatur similique, adipisci, debitis hic commodi impedit natus exercitationem tempore error, ratione dolore aliquam quis. Officia officiis veniam commodi.',
+    linkedinUrl : ''
+  }
+]
+
+currentMemberIndex = 0;
+
+nextMember() {
+  this.currentMemberIndex = (this.currentMemberIndex + 1) % this.members.length;
+}
+
+previousMember() {
+  this.currentMemberIndex = (this.currentMemberIndex - 1 + this.members.length) % this.members.length;
+}
+
+get whichMember () {
+  return (this.currentMemberIndex + 1) / this.members.length*100;
+}
+
+closeMemberView() {
+  console.log('return to about page')
```

Comment:
remove code that is not been used

Reply:
It is used when you want to close the view to go back to the team member carousel